contents/blog/mykonos-hackathon.md

```diff
@@ -0,0 +1,153 @@
+---
+date: 2024-05-28
+title: What we built at our Mykonos hackathon
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author:
+  - james-temperton
+featuredImage: >-
+  https://res.cloudinary.com/dmukukwp6/image/upload/posthog.com/contents/images/blog/aruba/beach-hog.jpg
+featuredImageType: full
+category: Inside PostHog
+tags:
+  - Offsites
+---
+
+Each year, the PostHog team gets together for our all-company offsite. This year, we headed to Mykonos – not to rave, but to code. And to enjoy a cocktail or two in the sun, of course.
+
+We’re a fully-remote company, with 47 misfits spread across ten countries. Our offsites are a really important part of our culture. They’re a great chance to get to know colleagues better and the connections formed at this time bring extra energy and creativity to our asynchronous work throughout the rest of the year.
+
+Our all-company offsites are a mix of socializing, group activities, strategic sessions, Post-its, workshops, more Post-its, and, the star of the show, the annual PostHog hackathon. Everyone in the company has to pitch a couple of ideas, then we all vote on our favorites, assemble teams, and have a little over a day to go from pitch to demo. And, in some cases, our hackathon projects are ready to ship right away.
+
+Here’s what we built in Mykonos.
+
+### The presidential briefing
```

Comment:
No offence to any particular teams here, but I think it'd be best if we can start off the list with something either very visually impactful (so we can have a cool big image), or technically cool to engineers. Something that can really start the list off with a bang: 10x Terms or Hog/Mykologs maybe.